tests/js/sentry-test/dropdownMenu.tsx

```diff
@@ -0,0 +1,92 @@
+import {act} from 'react-dom/test-utils';
+import {ReactWrapper} from 'enzyme'; // eslint-disable-line no-restricted-imports
```

Comment:
Not too sure what to do about this. `enzyme` direct imports are restricted. I could either disable the rule here (since it's just a type import), or just not import `ReactWrapper` and type `wrapper` (below) as `any`.

Reply:
I think that's fine in this case. 

Should we make it clear that this should only be used in enzyme tests? Maybe updating the file title or adding a deprecated tag with a description?